mongomock/collection.py

```diff
@@ -721,7 +721,10 @@ def _discard_operators(self, doc):
     def find(self, filter=None, projection=None, skip=0, limit=0,
              no_cursor_timeout=False, cursor_type=None, sort=None,
              allow_partial_results=False, oplog_replay=False, modifiers=None,
-             batch_size=0, manipulate=True):
+             batch_size=0, manipulate=True,
+             collation=None, hint=None, max_scan=None, max_time_ms=None,
```

Comment:
Hi @david1asher, it looks like there is no actual handling of the new args added here... Take a look [here](https://github.com/mongomock/mongomock#important-note-about-project-status--development) regarding our approach to stubs. Is there any way we can add the actual functionality here?

Reply:
Will investigate further about each arg implementation, thanks for the link